src/cunumeric/index/putmask_template.inl

```diff
@@ -105,7 +105,8 @@ static void putmask_template(TaskContext& context)
 {
   auto& inputs = context.inputs();
   PutmaskArgs args{context.outputs()[0], inputs[1], inputs[2]};
-  double_dispatch(args.input.dim(), args.input.code(), PutmaskImpl<KIND>{}, args);
+  int dim = args.input.dim() == 0 ? 1 : args.input.dim();
```

Comment:
I assume this PR also contains changes from #980, right?

Reply:
yes